roles/icpc_host_packages/tasks/main.yml

```
@@ -0,0 +1,116 @@
+---
+- name: Enable DNS bits in resolved.conf
+  ansible.builtin.replace:
+    path: /etc/systemd/resolved.conf
+    regexp: '^[#]?DNS=.*'
+    replace: 'DNS=8.8.8.8 8.8.4.4'
```

```suggestion
    replace: 'DNS=8.8.8.8 1.1.1.1'
```

In the unlikely situation that google has issues.

---

changed.